src/react/truststore/Truststore.tsx

```diff
@@ -17,31 +18,66 @@ import { Button } from '@scality/core-ui/dist/next';
 import { useBasenameRelativeNavigate } from '@scality/module-federation';
 import { useMemo } from 'react';
 import { MutationOptions, useQuery, useQueryClient } from 'react-query';
+import { CoreUIColumn, Row } from 'react-table';
 import { useToggleTLSVerificationMutation } from '../../js/mutations';
 import { ApiError } from '../../types/actions';
 import { getZenkoCRQuery } from '../queries';
 import { TableHeaderWrapper } from '../ui-elements/Table';
-import { CoreUIColumn, Row } from 'react-table';
+import { useParseBundleCertificates } from './hooks';
 
-type CertificateData = {
-  metadata: string;
-  expireOn: string;
-  authority: string;
+const formatCertificateDataForTable = (
+  parsedCertificates: (Certificate & CertificateSubject)[][],
```

Comment:
I have exposed the type `ParsedCertificate` that we could use directly. 
```suggestion
  parsedCertificates: ParsedCertificate[][],
```